velox/common/encode/tests/Base64Test.cpp

```diff
@@ -61,7 +64,7 @@ TEST_F(Base64Test, calculateDecodedSizeProperSize) {
   encoded_size = 21;
   EXPECT_THROW(
       Base64::calculateDecodedSize("SGVsbG8sIFdvcmxkIQ==", encoded_size),
-      facebook::velox::encoding::Base64Exception);
+      VeloxUserError);
```

Comment:
Let's use VELOX_ASSERT_THROW and validate the error message.

Reply:
done